### PR TITLE
feat : Add Type, Id, and Relationship annotations in models

### PR DESCRIPTION
**Pull request by Shailesh351**
Fixes : Part of #1680 

Changes: 
- Added jsonapi-converter dependency
- Added Type, Id, and Relationship annotations in models
- Added JsonSetter for jsonapi in copyright, CallForPapers

### CODE REVIEW

**Comment by Shailesh351** [comment]
@iamareebjamal @arp95 @shreyasnbhat @rohanagarwal94 Please review

File: android/app/src/main/java/org/fossasia/openevent/data/extras/CallForPapers.java [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
 
 import io.realm.RealmObject;
 import io.realm.annotations.PrimaryKey;
 
+@Type("speakers-call")
 public class CallForPapers extends RealmObject {
 
     @PrimaryKey
+    @Id(IntegerIdHandler.class)
+    private Integer id;
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/data/extras/CallForPapers.java]: Use primitive type instead

**Review by iamareebjamal — CHANGES_REQUESTED** [review]
Good Work!

**Review by iamareebjamal — APPROVED** [review]

**Review by arp95 — APPROVED** [review]
LGTM.

**Pull request merged**